lib/pick-fragment.glsl

```diff
@@ -1,11 +1,15 @@
 precision mediump float;
 
+#pragma glslify: outOfRange = require(glsl-out-of-range)
+
 uniform vec3  clipBounds[2];
 uniform float pickId;
 
 varying vec3 f_position;
 varying vec4 f_id;
 
 void main() {
+  if (outOfRange(clipBounds[0], clipBounds[1], f_position)) discard;
```

Comment:
doesn't look like this shader was doing *any* clipping before? Not even using `clipBounds`? So what changed that we need it now?

Reply:
With Etienne we had this discussion since it may help improve performance. And he suggested that I implement the same checks that we already have in other modules in cone3d & streamtube3d as well. If I recall correctly in certain interactive test scenarios with reversed ranges the pick displayed data points although there was no surface presented in the view.

Reply:
Good to know! Don't do it now, but why don't you make a `maintenance` issue to write a test for this kind of improper display & picking in each of our 3D trace types.